operators/multiclusterobservability/manifests/base/alertmanager/alert_rules.yaml

```
@@ -30,7 +30,7 @@ data:
             cluster: "{{ $labels.cluster }}"
             clusterID: "{{ $labels.clusterID }}"
             PersistentVolumeClaim: "{{ $labels.persistentvolumeclaim }}"
-            severity: warning
+            severity: informational
```

An alert severity level for `informational` is `info`:
```suggestion
            severity: info
```

---

done